scripts/cipublish

```diff
@@ -31,6 +31,9 @@ then
         # Mastarm doesn't copy over index.html
         aws s3 cp taui/index.html "${TAUI_SITE_BUCKET}/index.html"
 
+        # Move the Amplify config for the proper env into the source dir
+        cp "deployment/amplify/${env}/aws-exports.js" taui/src/aws-exports.js
```

Comment:
This is necessary to bundle the Amplify exports into the app for the appropriate environment.